Python/codegen.c

```diff
@@ -4010,6 +3996,10 @@ codegen_call_helper(compiler *c, location loc,
         assert(elt->kind != Starred_kind);
         VISIT(c, expr, elt);
     }
+    if (injected_arg) {
```

Comment:
You also need to do something similar in the cases where we `goto ex_call`.

With your PR:

```
>>> class A[T](*()): pass
... 
>>> A.__bases__
(<class 'object'>,)
```

Reply:
are we missing a test for this?

Reply:
Yes, it would be good to have more test coverage here.

Reply:
Ok, I'll continue tomorrow.